codalab/server/oauth2_provider.py

```diff
@@ -151,7 +150,7 @@ def validate_client_id(self, client_id):
 
             oauth._validator = MyValidator()
         """
-        expires_in = self.app.config.get('OAUTH2_PROVIDER_TOKEN_EXPIRES_IN', parse_duration("30d"))
+        expires_in = self.app.config.get('OAUTH2_PROVIDER_TOKEN_EXPIRES_IN')
```

Comment:
@teetone make an issue for this --

try making the access token lifetime really short (15 secs), see if the refresh token works.